src/components/BooksForm.js

```diff
@@ -33,25 +33,36 @@ function Form() {
   };
 
   return (
-    <form className="FORM" onSubmit={handleSubmit}>
-      <input
-        type="text"
-        id="title"
-        name="title"
-        required
-        value={state.title}
-        onChange={onChangeHandler}
-      />
-      <input
-        type="text"
-        id="author"
-        name="author"
-        required
-        value={state.author}
-        onChange={onChangeHandler}
-      />
-      <button type="submit">Add Book</button>
-    </form>
+    <div className="Add-container">
+      <h2 className="add-title">Add Your Book</h2>
+      <section>
+        <form className="add-book" onSubmit={handleSubmit}>
+          <input
+            type="text"
+            className="title-input"
+            id="title"
+            placeholder="Add Book Title"
+            name="title"
+            required
+            value={state.title}
+            onChange={onChangeHandler}
+          />
+          <input
+            type="text"
+            id="author"
+            className="author-input"
+            name="author"
+            placeholder="Add Book Author"
+            required
+            value={state.author}
+            onChange={onChangeHandler}
+          />
+          <button className="add-button" type="submit">
+            Add Book
+          </button>
+        </form>
+      </section>
```

Comment:
- [ ] 👍 You're on the right track with your form, but there's a layout adjustment needed to align it with the design. Please consider updating your layout to match the design specifications. This will ensure that your form not only looks good but also aligns perfectly with the intended design. Keep up the great work! 👍📝

| Your | Design |
| --- | --- |
|  ![Frame 5 (1)](https://github.com/Nuga0909/bookstore3/assets/55840999/b5aa23c8-ed82-4f93-a545-1bad4efa025c)| ![image](https://github.com/Nuga0909/bookstore3/assets/55840999/00cd287b-759d-49ba-88f2-a0dd2433d682)|